net-misc/keychain/files/keychain-2.8.5-empty-ssh-askpass.patch

```diff
@@ -0,0 +1,19 @@
+Description: Handle an empty (or unset) SSH_ASKPASS variable
```

Comment:
I'm going to include ilnks to where the patch came from at the top of each patch, per https://devmanual.gentoo.org/ebuild-maintenance/git/index.html#configuration-specific-to-the-gentoo-repository.